skll/learner.py

```diff
@@ -869,9 +870,13 @@ class Learner(object):
         initializer for the specified model.
         Defaults to ``None``.
     pos_label_str : str, optional
```

Comment:
Is it true that this will necessarily be a `str` here? I'm a little confused because it seems like it could be a float or int at this point too (in addition to `None`).

Reply:
Yes, it could be anything but it's easier to say `str` because even if the user puts `"1"` or `"1.0"` or anything, `safe_float()` will handle it just fine. So, I didn't want to make the docstring overly complicated.

Reply:
I'm mainly concerned with the documentation. If we were to use a type annotation, this would probably need to be changed to be `Optional[Union[str, float, int]]`.

Reply:
Sure, we can do that when it comes time to do that.